data-collection/deploy/deploy-in-management-account.yaml

```diff
@@ -365,4 +374,41 @@ Resources:
       cfn_nag:
         rules_to_suppress:
           - id: W12
-            reason: "Policy is used for scanning of a wide range of resources"
\ No newline at end of file
+            reason: "Policy is used for scanning of a wide range of resources"
+  # Marketplace Agreements policy
```

Comment:
do we need it in management account? there is an option to deploy linked acc stack in mngment.